modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/NPMRegistry.java

```diff
@@ -45,27 +45,52 @@
 import org.osgi.util.tracker.BundleTrackerCustomizer;
 
 /**
+ * <p>
+ * This is the central class for the NPM packages support of Liferay Portal.
+ * </p>
+ * <p>
+ * It features a central registry where all NPM packages and modules deployed
+ * accross OSGi bundles are tracked.
+ * </p>
```

Comment:
@izaera what do you mean when saying

> ...where all NPM packages and modules deployed across OSGi bundles are tracked.

I'm stumbling on the word **across**.  Can we say that the NPM packages and modules deployed **with** OSGi modules are tracked?